requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+mypy==1.8.0; implementation_name=="cpython"
 
-pre-commit==3.6.0
+pre-commit>=3.5
```

Comment:
Requirements file should be pinned to specific versions. Dependabot will update them.

Reply:
3.6.0 is not compatible with 3.8 :(
But if I develop this project with newer versions - mypy asks to use new generic types
How do you manage this?

Reply:
pre-commit is only used for the linter step, it doesn't need to work with Python 3.8 (which is why the CI is passing already with this). Just run it on a more recent version of Python.

Reply:
> But if I develop this project with newer python versions - mypy asks to use new generic types

Not sure which you are referring to? We don't have problems in our other repos (unless there was a change in the last release I've not looked at yet).

Reply:
https://github.com/aio-libs/aiohttp-sse/pull/441#discussion_r1468888938